api/services/MachineService.js

```diff
@@ -508,20 +508,21 @@ function updateMachinesPool() {
 function _shouldTerminateMachine(machine) {
   Promise.props({
     isActive: machine.isSessionActive(),
-    config: ConfigService.get('neverTerminateMachine')
+    config: ConfigService.get('neverTerminateMachine'),
+    machineToTerminate: Machine.findOne({id: machine.id})
```

Comment:
`machine` and `Machine.findOne(machine)` are strictly the same. How is this fixing something ?